regression/cbmc/malloc-too-large-almost/main.c

```diff
@@ -0,0 +1,13 @@
+#include <assert.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+int main()
+{
+  size_t pointer_width = sizeof(void *) * 8;
+  size_t object_bits = 8; // by default
+  size_t cbmc_max_alloc_size = (size_t)1 << (pointer_width - object_bits);
```

Comment:
A really tiny nit pick, so feel free to ignore me :-) But I'd personally write this as:
```
size_t cbmc_max_alloc_size = ((size_t)1 << (pointer_width - object_bits)-1);
int *p2 = malloc(cbmc_max_alloc_size); // try to allocate exactly the max
```
just so that the variable names really do mean what they say. Ditto for the other test case - but like I say, it's a really minor nit for me to pick...